### PR TITLE
Update marine-js docs

### PR DESCRIPTION
**Pull request by ValeryAntopol**

### CODE REVIEW

File: docs/marine-book/marine-runtime/api/marine-js-api.md [hunk 1]
```diff
@@ -1,72 +1,126 @@
 # Marine-JS API
 
-Web runtime is support not all features of the Rust Marine side, it supports now only pure single-module services. `FluenceAppService` is the central pillar of the web runtime, it could be reflected to the Runtime component despite its name. In the future on the milestone 4 of the web runtime JS `FluenceAppService` will have a one-to-one correspondence with Rust one.
+Web runtime does not support not all features of the Rust Marine side, it supports now only pure services and does not pass call parameters to the service, but supports multi-module services. `MarineService` is the central pillar of the web runtime, it has simplified interface of `FAppService`. In the future on the milestone 4 of the web runtime JS `MarineService` will have a one-to-one correspondence with Rust one.
```
Comment (mikevoronov) [inline docs/marine-book/marine-runtime/api/marine-js-api.md]:
```suggestion
Web runtime supports all features of the Rust Marine except a part of WASI features. `MarineService` is the central pillar of the web runtime, it corresponds to `FluenceAppService` from the Rust side but has a simplified interface. In the future, with milestone 4 `MarineService` will have a one-to-one correspondence with the Rust side.
```

File: docs/marine-book/marine-runtime/api/marine-js-api.md [hunk 2]
```diff
@@ -1,72 +1,126 @@
...
-    Promise<SharedArrayBuffer | Buffer>
 ```
-
-Loads WASM file from the service hosting the application. It accepts the file path on the server and returns a buffer compatible with `FluencePeer` API. This function can only be used in a browser. Trying to call it inside node.js will result throw an error.
-
-## Playing with FluenceAppService
-
-### initialization
-
+A marine service can produce logs during execution. All of its logs will be passed to the `logFunction` argument, which has the following type:
```
Comment (mikevoronov) [inline docs/marine-book/marine-runtime/api/marine-js-api.md]:
```suggestion
A marine service can produce logs during execution. All of these logs will be passed into the `logFunction` function, which has the following type:
```

File: docs/marine-book/marine-runtime/api/marine-js-api.md (hunk 1 above)
Comment (boneyard93501) [reply docs/marine-book/marine-runtime/api/marine-js-api.md]: The web runtime is closely but not completely feature equivalent with the Rust Marine runtime. Specifically, the following WASI features ???, ??? and ??? are not supported which means ???.

[MarineService]() is the central pillar of the web runtime and corresponds to [FluenceAppService]() from the Rust runtime. Currently,  `MarineService` provides a simpler interface than `FluenceAppService` but matched correspondence is expected as part of [MVM]().

maybe link to `MarineService`  and `FluenceAppService`

File: docs/marine-book/marine-runtime/api/marine-js-api.md (hunk 2 above)
Comment (boneyard93501) [reply docs/marine-book/marine-runtime/api/marine-js-api.md]: Logs generated by a marine service are passed to the `logFunction`, which has the following type:

**Comment by boneyard93501** [comment]
nice. starting to shape up pretty well.

File: docs/marine-book/marine-runtime/api/marine-js-api.md (hunk 1 above)
Comment (ValeryAntopol) [reply docs/marine-book/marine-runtime/api/marine-js-api.md]: Added links to the `MarineService` and `FluenceAppService` and fixed WASI stuff, but we decided to dismiss the part about MVM and just say that the interfaace correspondence is planned

**Review by mikevoronov — APPROVED** [review]
utACK

**Pull request merged**